src/auth/screens/auth-profile.screen.js

```diff
@@ -113,34 +115,68 @@ class AuthProfile extends Component {
     }
   };
 
+  refreshProfile = () => {
+    this.props.getUser();
+    this.props.getOrgs();
+  };
+
   render() {
     const { user, orgs, isPendingUser, isPendingOrgs, navigation } = this.props;
     const loading = isPendingUser || isPendingOrgs;
 
     return (
       <ViewContainer>
-        {loading && <LoadingContainer animating={loading} center />}
-
-        {!loading &&
-          <ParallaxScroll
-            renderContent={() =>
-              <UserProfile
-                type="user"
-                initialUser={user}
-                user={user}
-                navigation={navigation}
-              />}
-            stickyTitle={user.login}
-          >
-            {user.bio &&
-              user.bio !== '' &&
-              <SectionList title="BIO">
+        <ScrollView
+          refreshControl={
+            <RefreshControl
+              refreshing={loading}
+              onRefresh={this.refreshProfile}
+            />
+          }
+        >
+          {loading && <LoadingContainer animating={loading} center />}
+
+          {!loading &&
+            <ParallaxScroll
+              renderContent={() =>
+                <UserProfile
+                  type="user"
+                  initialUser={user}
+                  user={user}
+                  navigation={navigation}
+                />}
+              stickyTitle={user.login}
+            >
+              {user.bio &&
+                user.bio !== '' &&
+                <SectionList title="BIO">
+                  <ListItem
+                    subtitle={user.bio}
+                    subtitleStyle={styles.listSubTitle}
+                    hideChevron
+                  />
+                </SectionList>}
+
+              <SectionList
+                title="EMAIL"
+                noItems={!user.email || user.email === ''}
+                noItemsMessage={'No email associated with account'}
+              >
                 <ListItem
-                  subtitle={user.bio}
+                  title="Email"
+                  titleStyle={styles.listTitle}
+                  leftIcon={{
+                    name: 'mail',
+                    color: colors.grey,
+                    type: 'octicon',
+                  }}
+                  subtitle={user.email}
                   subtitleStyle={styles.listSubTitle}
-                  hideChevron
+                  onPress={() =>
+                    Communications.email([user.email], null, null, 'Hi!', '')}
+                  underlayColor={colors.greyLight}
                 />
-              </SectionList>}
+              </SectionList>
```

Comment:
Yeah a bit of a mistake when merging...  Last commit should have removed it I think, a second swing at merging your master into my branch